addons/web/static/src/core/commands/command_palette.js

```diff
@@ -100,6 +100,14 @@ export class CommandPalette extends Component {
             commands: [],
         });
 
+        onWillUnmount(() => {
+            for (const provider of this.props.config.providers) {
+                if (provider.onClose) {
+                    provider.onClose();
```

Comment:
@brboi @FrancoisGe Hello ! I'd like to check with you for validating two names we'd like to add here:
- The `shortcut_conflict` command category which has the lowest sequence so that the associated command is always the first suggestion.
- The `onClose` provider function to execute code after the command palette is closed.

Do you think these names make sense in the way you envisioned the commands categories and providers, or would you propose alternative names ? Thanks !

Reply:
FYI we discussed with NBY this morning and we agreed that `shortcut_conflict` should get added only when the web_editor addon is installed and we also provided an alternative to the `onClose` thing: the command palette would somehow instruct the providers that it's the first time they're being called (i.e. by providing a command palette session id).

Reply:
Thanks for letting me know. 👍